### PR TITLE
[10.0][IMP] account_credit_control use partner email

### PR DESCRIPTION
**Pull request by i-vyshnevska**
when no any email avaible on invoice contract use
parent email if exists

### CODE REVIEW

**Review by TDu — APPROVED** [review]

**Review by pedrobaeza — COMMENTED** [review]
Why not using `contact.partner_commercial_id`?

**Comment by i-vyshnevska** [comment]
> Why not using `contact.partner_commercial_id`?

@pedrobaeza thanks for pointing, updated

File: account_credit_control/wizard/credit_control_communication.py [hunk 1]
```diff
@@ -89,7 +89,13 @@ def get_email(self):
         """ Return a valid email for customer """
         self.ensure_one()
         contact = self.contact_address
-        return contact.email
+        email = contact.email
+        if (
+            not email and contact.commercial_partner_id
```
Comment (pedrobaeza) [inline account_credit_control/wizard/credit_control_communication.py]: `and contact.commercial_partner_id` can be removed as redundant

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge patch

You can forward-port this to upper versions.

**Pull request merged**